src/cli_plugin/install/__tests__/settings.js

```diff
@@ -0,0 +1,230 @@
+import path from 'path';
+import expect from 'expect.js';
+
+const utils = require('requirefrom')('src/utils');
```

Comment:
We dropped requirefrom use, or will be very soon. Also, pretty sure we're only using import now.